### PR TITLE
allow group meta info

### PR DESCRIPTION
**Pull request by gaffa**
please review...

### CODE REVIEW

File: dash-core/src/main/java/de/axelspringer/ideas/tools/dash/presentation/UIGroup.java [hunk 1]
```diff
@@ -5,9 +5,7 @@
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 import org.apache.commons.lang3.builder.ToStringBuilder;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```
Comment (CanardSauvage) [inline dash-core/src/main/java/de/axelspringer/ideas/tools/dash/presentation/UIGroup.java]: Why did you do this? Most code guidlines see this as bad thing.

**Comment by CanardSauvage** [comment]
For what do you use the metaInfo?

File: dash-core/src/main/java/de/axelspringer/ideas/tools/dash/presentation/UIGroup.java (hunk 1 above)
Comment (gaffa) [inline dash-core/src/main/java/de/axelspringer/ideas/tools/dash/presentation/UIGroup.java]: the star import? my ide organizes the imports for me. i never look at them :)

**Comment by gaffa** [comment]
We use metaInfo to display build parameters on pipeline execution.
Or at least planned to. Currently I am evaluating other visualization options, too...

**Pull request merged**